platform/srf06-cc26xx/contiki-main.c

```diff
@@ -123,6 +126,10 @@ set_rf_params(void)
     printf("%02x\n", linkaddr_node_addr.u8[i]);
   }
 #endif
+
+  /* also set the global node id */
+  node_id = short_addr;
```

Comment:
We discussed this with George off-line and he's ok with this change. I personally think it would be nice to have `node_id` for all Contiki platforms. There are other examples besides `rpl-tsch` (at least one even in Contiki mainline) that rely on `node_id`. A nice way select address-specific functionality in platform and even network stack independent way (works with Rime too).